models/user.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema; // Used to create a new Schema
+
+// Define the Schema for user
+// TODO we can check for validation in the future
```

Comment:
For validation and the next CR can you connect one of the GET endpoint to get any data from the database

Reply:
Resolved